presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -335,6 +351,22 @@ public PlanNode visitRemoteSource(RemoteSourceNode node, RewriteContext<Void> co
         }
 
         @Override
+        public PlanNode visitMergeRemoteSource(MergeRemoteSourceNode node, RewriteContext<Void> context)
```

Comment:
Yes. I've got a test failures because of this. I need to canonicalize ordering symbols at this step.